static/js/accessibility/agencies.js

```diff
@@ -1,59 +1,20 @@
 $(document).ready(function () {
 
-  $.get("/static/data/tables/accessibility/agencies.json", function(data) {
-    renderTable(data.data);
-  });
-
-  var renderTable = function(data) {
-    var table = $("table").DataTable({
-      responsive: true,
-      initComplete: Utils.searchLinks,
-
-      data: data,
+  // break caching
+  var time = (new Date()).getTime();
 
+  $.get("/static/data/tables/accessibility/agencies.json?" + time, function(data) {
+    Tables.initAgency(data.data, {
```

Comment:
The `Tables.initAgency` method is now defined centrally, and handles most common configuration of DataTables across all our pages.